src/test/java/uk/nhs/digital/nhsconnect/lab/results/IntegrationAdapterLabResultsApplicationTests.java

```diff
@@ -2,7 +2,9 @@
 
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.scheduling.annotation.EnableScheduling;
 
+@EnableScheduling
```

Comment:
This shouldn't be needed here.

Reply:
true. Fixed in 7dd188ba0264bd8b9446d2b1480e976767e56204 . Thank you!